inst/templates/Makefile

```diff
@@ -0,0 +1,29 @@
+R = R --vanilla CMD
+
+
+all: build check install clean
```

Comment:
I don't really think install should be in `all`, maybe just `check` (which implies `build`)?